services/api/src/utils/middleware/validate.js

```diff
@@ -1,82 +1,87 @@
-function validate(schemas, options) {
-  const defaultOptions = {
-    allowUnknown: false,
-    abortEarly: false,
-  };
+const Joi = require('joi');
 
-  const promise = async function Validate(ctx, next) {
-    Object.keys(schemas).forEach((key) => {
-      const schema = schemas[key];
+const DEFAULT_OPTIONS = {
+  allowUnknown: false,
```

Comment:
This may have been part of previous PR, but it it described somewhere in the readme's on what these options mean

Reply:
these are the default options to Joi for validating. They're not documented but on the other hand they're not new. They've always been there I just moved and renamed... for my part it's pretty clear what they do when I look at them as they're well named so I never questioned them.